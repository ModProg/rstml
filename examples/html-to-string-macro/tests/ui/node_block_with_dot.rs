use html_to_string_macro::html;
fn main () {
    html! {
        <!DOCTYPE html>
            <html>
            {"block".}
            </html>
    };
}