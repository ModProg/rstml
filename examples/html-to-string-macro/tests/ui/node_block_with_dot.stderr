error: unexpected end of input, expected identifier or integer
 --> tests/ui/node_block_with_dot.rs:6:22
  |
6 |             {"block".}
  |                      ^

error: unexpected token: `}`
 --> tests/ui/node_block_with_dot.rs:6:13
  |
6 |             {"block".}
  |             ^^^^^^^^^^
